Added documentation for the alert in the login page (#1189)
uyuni-project/uyuni-docs: modules/administration/pages/space-management.adoc

You can modify which directories are monitored, and the warnings that are created.
All settings are configured in the [path]``/etc/rhn/rhn.conf`` configuration file.

> When the available space in one of these monitored directories falls below a warning threshold, an email is sent to the configured email address and a message is shown at the top of the login page.

Reviewer: ```suggestion
When the available space in one of the monitored directories falls below a warning threshold, a message is sent to the configured email address and a notofication is shown at the top of the sign-in page.
```

Author: There is a typo ("notofication") in your suggested change. If you edit it, I will commit it together with the other one.

systemctl stop spacewalk-diskcheck.timer
systemctl disable spacewalk-diskcheck.timer
----

> Doing this disables the periodic email alerts if the alert threshold is reached, but the warning message will still appear at the top of the login page.

Reviewer: ```suggestion
Disabling the [literal]``spacewalk-diskcheck.timer`` will stop periodic email alerts if the alert threshold is reached, but the warning notification will still appear at the top of the sign-in page.
```
